xml/Microsoft.Extensions.DependencyInjection/ServiceProviderOptions.xml

```diff
@@ -99,9 +98,9 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>
-          <see langword="true" /> to perform check verifying that scoped services never gets resolved from root provider; otherwise, <see langword="false" />. Defaults to <see langword="false" />.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets a value that indicates whether validation is performed to ensure that scoped services are never resolved from the root provider.</summary>
+        <value>
+          <see langword="true" /> if validation is performed to ensure that scoped services aren't resolved from the root provider; otherwise, <see langword="false" />. The default is <see langword="false" />.</value>
```

Comment:
```suggestion
          <see langword="true" /> if validation is performed to ensure that scoped services aren't resolved from the root provider; otherwise, <see langword="false" />. The default is the value of <see cref="M:Microsoft.Extensions.Hosting.HostingEnvironmentExtensions.IsDevelopment(Microsoft.Extensions.Hosting.IHostingEnvironment)" />.</value>
```